synfig-core/src/synfig/valuenode.cpp

```diff
@@ -800,6 +800,18 @@ LinkableValueNode::set_root_canvas(etl::loose_handle<Canvas> x)
 		get_link(i)->set_root_canvas(x);
 }
 
+LinkableValueNode::InvertibleStatus LinkableValueNode::is_invertible(Time /*t*/, const ValueBase& /*target_value*/, int* link_index) const
```

Comment:
The purpose of `link_index` was completely unclear to me, when I see it first time.
So maybe add a little comment about it?

Reply:
It's on .h :(

Reply:
Sorry, I don't know how I missed it :)

Reply:
TOo manY CoDe to read ;)